### DISCUSSION TITLE
Speed and stability

### DISCUSSION THREAD

**Discussion by davehakkens** [ONEARMY/community-platform#3239] [Big Updates]
# Current problems
1. Concerns has been flagged about the speed of the platform
2. Concerns have been flagged about the stability of the platform

# What is needed:

- [ ] Research and identify pages with long loading times and document what makes them slow.
- [ ] Find a way to analyse the stability of the platform
- [ ] Create a technical plan to address the issues

**Comment by davehakkens** [comment]
hey @mariojsnunes perhaps you have some ideas on this? (also send you a DM in Discord, not sure if you get notifications)

**Reply by mariojsnunes** [reply]
Would need to do further testing to confirm, but I guess the most likely cause is that we are getting the entire collection, then filtering/sorting on the frontend.
The work I've been doing here should improve things. https://github.com/ONEARMY/community-platform/pull/3222
I want to finish it today, then evaluate and think about migrating other modules.
The code is also simpler so should reduce bugs.
